### ISSUE TITLE
Plot uncertainty as function of training set size

### ISSUE THREAD

**Issue by nbgl** [chengsoonong/mclass-sky#188]
Include the mean and the standard deviation of the uncertainties in the test set. Cheng wants this done for GPs, but it may also be a good idea to do for approximate GPs.

**Comment by nbgl** [comment]
This has not been done, but it is one of the plots I want to include in thesis.
